scripts/menu.js

```diff
@@ -0,0 +1,20 @@
+const menuBtn = document.getElementById('menu-toggle');
+const nav = document.querySelector('nav');
```

Comment:
- [ ] Well done @astroboyReloaded, you have at least created the toggle functions but .... Either it is not how you should render your projects, I remark the first issue is 

1. Nothing happens when you click the `see project button` button. fix this issue as we go
2. You are still using your static project data on the HTML page, see ⬇️ 
<img width="403" alt="1" src="https://user-images.githubusercontent.com/77843050/215824183-088f50ea-37a5-418a-ac07-376c34d9290a.png">

but referring to the project requirement below, it is now how you should present the projects, instead, they should be dynamic(creating using js, and rendering on page load)

<img width="1075" alt="2" src="https://user-images.githubusercontent.com/77843050/215824076-bbb45a8c-269c-4800-9be1-46a71e0c71ad.png">

- Pease ensures you remove all project static codes on the page and use js array to store your project data.


- [ ] THIS IS THE RECORD

I have made sure to clone your project as well as use the online version checking, but still the same thing

![5](https://user-images.githubusercontent.com/77843050/215826672-ee97fb90-7e46-4028-a11b-1696ffe18da3.gif)

Reply:
Sorry, @joskalenda. I submitted the project to the wrong Request. 😕 
Thank you, though!